src/exchanges.js

```diff
@@ -297,3 +317,23 @@ exchanges.declare({
   routingKeyBuilder:  commonRoutingKeyBuilder,
   CCBuilder:          commonCCBuilder,
 });
+
+/** Task group resolved exchange */
+exchanges.declare({
+  exchange:           'task-group-resolved',
+  name:               'taskGroupResolved',
+  title:              'Task Group Resolved',
+  description: [
+    'Whenever a task group has run to completion (success is not considered),',
+    'a message will be sent on this exchange. It can be listened to to know',
+    'when a group is "complete". There is no guarantee that this group is now',
+    'intert. It can have more tasks added to it by the scheduler, but at least',
```

Comment:
@imbstack What is intert? I guess this is a typo, but I can't work out what word should be there. :-)

Inert?

Also, I thought we were ditching the scheduler - are tasks not added now by the queue?